engine/language-server/src/test/scala/org/enso/languageserver/libraries/LocalLibraryManagerSpec.scala

```diff
@@ -0,0 +1,74 @@
+package org.enso.languageserver.libraries
+
+import akka.actor.ActorSystem
+import akka.testkit._
+import org.enso.distribution.FileSystem.PathSyntax
+import org.enso.editions.LibraryName
+import org.enso.librarymanager.LibraryLocations
+import org.enso.pkg.PackageManager
+import org.enso.testkit.WithTemporaryDirectory
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpecLike
+import org.scalatest.BeforeAndAfterAll
+import org.scalatest.time.SpanSugar.convertIntToGrainOfTime
+
+import scala.concurrent.duration.FiniteDuration
+
+class LocalLibraryManagerSpec
+    extends TestKit(ActorSystem("TestSystem"))
+    with ImplicitSender
+    with AnyWordSpecLike
+    with Matchers
+    with BeforeAndAfterAll
+    with WithTemporaryDirectory {
+
+  val Timeout: FiniteDuration = 10.seconds
+
+  override def afterAll(): Unit = {
+    TestKit.shutdownActorSystem(system)
+  }
+
+  "LocalLibraryManager" should {
+    "find the libraries it has itself created" in {
```

Comment:
This test is partially redundant with https://github.com/enso-org/enso/blob/5dc2c4c5fd4a2da97434ea83983361d3f8ef493e/engine/language-server/src/test/scala/org/enso/languageserver/websocket/json/LibrariesTest.scala#L55 but it checks some more cases and is more a 'unit test' whereas the other is more an 'integration test'. 

So IMO it may make sense to keep both - the role of the newly added one is to ensure the logic for creating a library is kept in-sync with the logic for resolving them. The other test is just an integration test checking library management.